graphalyzer-server/src/app.py

```diff
@@ -36,7 +36,7 @@ def load_edges(graphlocation: str, graphid: str):
     from py2neo import Graph
     from py2neo.packages.httpstream import http
     # Increase timeout because it can take a really long time.
-    http.socket_timeout = sys.maxsize
+    http.socket_timeout = 9999
```

Comment:
Which we can't do much about, unless there's a more graceful way of handling that situation, but that's a separate issue.

So all good from me. :+1: